tests/test_modelhub_cli.py

```diff
@@ -53,20 +58,18 @@ def test_update():
     assert result.exit_code == 0
 
 
-def test_delete():
-    with requests.get(f'{app_settings.api_v1_prefix}/model/') as r:
-        model_list = r.json()
-    model_id = model_list[0]["id"]
-    result = runner.invoke(app, ['delete', model_id])
-    assert result.exit_code == 0
-    assert f"Model {model_id} deleted\n" == result.output
-
-
 def test_convert():
-    torch_model = torchvision.models.resnet50(pretrained=False)
-    torch_model.load_state_dict(torch.load(file_path))
-    torch.save(torch_model, file_path)
     result = runner.invoke(app, [
         'convert', '-f', 'example/resnet50.yml'
     ])
-    assert result.exit_code == 0
\ No newline at end of file
+    assert result.exit_code == 0
+
+
+def test_delete():
+    with requests.get(f'{app_settings.api_v1_prefix}/model/') as r:
+        model_list = r.json()
+    for model in model_list:
+        model_id = model["id"]
+        result = runner.invoke(app, ['delete', model_id])
+        assert result.exit_code == 0
+        assert f"Model {model_id} deleted\n" == result.output
```

Comment:
why the output is a string?

Reply:
Here the `result` variable is a `click.testing.Result` instance, its `output` property will return a string which represent the standard output, refer to https://click.palletsprojects.com/en/8.0.x/api/#click.testing.Result.output